jerry-ext/include/jerryscript-ext/arg.impl.h

```diff
@@ -347,8 +347,9 @@ jerryx_arg_custom (void *dest, /**< pointer to the native argument where the res
  * @return a jerryx_arg_t instance.
  */
 static inline jerryx_arg_t
-jerryx_arg_object_properties (const jerryx_arg_object_props_t *object_props, /**< pointer to object property mapping */
-                              jerryx_arg_optional_t opt_flag) /**< whether the argument is optional */
+jerryx_arg_object_properties (
```

Comment:
What do you think about renaming to `obj_prop_p` instead, so it can be written in the same line. This indentation is a bit unusual.